assets/css/style.css

```diff
@@ -309,3 +307,122 @@ hr {
   text-align: center;
   color: #6070ff;
 }
+
+@media (min-width: 768px) {
+  /* header */
+  header {
+    height: 73px;
+  }
+
+  header .container img {
+    display: none;
+  }
+
+  header .container ul {
+    display: flex;
+    gap: 20px;
+    color: #344563;
+  }
+
+  #hero {
+    background: url("../images/Header\ bg.svg") no-repeat #fff 100%;
+    border-bottom-left-radius: 160px;
+    background-position: center;
+    background-size: contain;
+    padding: 50px 140px;
+  }
+
+  /* card */
+  #work {
+    margin-top: 104px;
+  }
+
+  .work {
+    padding: 0 20px;
+  }
+
+  .card {
+    flex-direction: row;
+  }
+
+  .card .card-img {
+    width: 100%;
+    padding-right: 24px;
+  }
```

Comment:
- [OPTIONAL] Good job for reversing the cards as suggested by the reviewer 🥇🥇 Please consider fixing the width of the images in project cards to get a design close to the Figma template as much as possible 🤓🔨 One possible implementation can be 👇

```
.card .card-img {
  // other properties
  flex: 1.5;
}

content {
  flex: 2;
}
```